### PR TITLE
The great crystallization of golems

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
Gives 3 "crystalline" golems (diamond, bluespace and glass) an unique sprite. Also adds a new type of golem, a telecrystal golem that is created with 10 telecrystals.

obraz

males are more spiky, females are more blocky

### Why is this change good for the game?
Makes those golems more unique, people often mistook diamond golems for adamantine golems.
Adds a new feature: a telecrystal golem with it's unique ability to phase jump!
Also makes it possible to actually create metal hydrogen golems.

# Wiki Documentation
### Briefly describe your PR and the impacts of it, in layman's terms. 
Golems that are made of crystalline objects have an unique appearance.
The new telecrystal golem, made with 10 telecrystals, has the usual armor for a golem, speed of a gold golem - slower than human, faster than most golems, and comes with an unique ability: Phase Jump. This ability lets the golem teleport to any visible place in the 3 tile radius around it, on a moderate cooldown, similarly to how the ekatana dash works. Likewise, this ability also makes a trail where the golem teleported, making it easily trackable.
For obvious reasons this golem isn't in the random golem pool.

### What should players be aware of when it comes to the changes your PR is implementing?
You can withdraw your TC to make a golem out of it.

### What general grouping does this PR fall under? 
idk, golem changes and additions?

# Changelog
:cl:  
rscadd: adds a telecrystal golem  
bugfix: makes it able to create metal hydrogen golems
tweak: gives 3 golems a new unique crystalline sprite
imageadd: added grayscale crystal golem bodyparts, added sprites for the new phase jump ability
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
image

**Comment by ChesterTheCheesy** [comment]
obraz

**Comment by PantsVance** [comment]
Yo this is pimp, I see no reason for this to not be in the game
